uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1092,17 +1092,22 @@ sub dicom_to_minc {
     my $this = shift;
     my ($study_dir, $converter,$get_dicom_info,
 		$exclude,$mail_user, $upload_id) = @_;
-    my ($d2m_cmd,$d2m_log,$exit_code);
+    my ($d2m_cmd, $d2m_log, $exit_code, $excluded_regex);
     my $message = '';
 
     # create the excluded series description regex necessary to exclude the
     # series description specified in the Config Setting
     # excluded_series_description
-    my $excluded_regex = join('|', map { quotemeta($_) } @$exclude);
+    if ($exclude && ref($exclude) eq 'ARRAY') {
+        $excluded_regex = join('|', map { quotemeta($_) } @$exclude);
+    } elsif ($exclude) {
+        $excluded_regex = $exclude;
+    }
     $d2m_cmd = "find $study_dir -type f | $get_dicom_info -studyuid -series".
                " -echo -image -file -series_descr -attvalue 0018 0024".
-               " -stdin | sort -n -k1 -k2 -k6 -k3 -k7 -k4 | grep -iv".
-               " -E \"($excluded_regex)\" | cut -f 5 | ";
+               " -stdin | sort -n -k1 -k2 -k6 -k3 -k7 -k4 ";
+    $d2m_cmd .= " | grep -iv -E \"($excluded_regex)\"" if ($excluded_regex);
```

Comment:
Just thought of this: the `grep` command will check for an occurence of any of the excluded series _anywhere_ on the string, not just in the series_descr value returned by the call to $get_dicom_info. Could this be a problem?

Also, if your study has two series described as, say 't1' and 't1_weighted', and you specify that you want to exclude serie 't1', then serie 't1_weighted' will also be excluded (as 't1_weighted` matches regex 't1')

Reply:
At first, I thought it would not be a bad idea so that anything that has Scout or Localizer in it can be removed from `dcm2mnc` (this was also the behavior before BTW).

However, I see your point about the t1 excluding other series descriptions. Not sure how to fix this though

Reply:
@nicolasbrossard I updated the PR so that the regex can have a regular expression that can look like something like that: `"AAHScout\s+$|AAHScout_MPR\s+$`. 
Unfortunately, I cannot use the `^` at the beginning of the each value of the regex as the regex is run on the output of `get_dicom_info.pl` which looks like this:
```
1.3.12.2.1107.5.2.32.35442.30000017021714191057800000010	18	1 	1 	./MR.1.3.12.2.1107.5.2.32.35442.2017021715254857655874003	*fl3d1_ns 	AAHScout_MPR	
1.3.12.2.1107.5.2.32.35442.30000017021714191057800000010	18	1 	2 	./MR.1.3.12.2.1107.5.2.32.35442.2017021715254858304274005	*fl3d1_ns 	AAHScout_MPR	
1.3.12.2.1107.5.2.32.35442.30000017021714191057800000010	18	1 	3 	./MR.1.3.12.2.1107.5.2.32.35442.2017021715254858817874007	*fl3d1_ns 	AAHScout_MPR	
1.3.12.2.1107.5.2.32.35442.30000017021714191057800000010	18	1 	4 	./MR.1.3.12.2.1107.5.2.32.35442.2017021715254859335574009	*fl3d1_ns 	AAHScout_MPR
```
See commit https://github.com/aces/Loris-MRI/pull/326/commits/423826946ba06b1c20604c98dea0f525fc0e1561

It was the best I could do. At least, the `$` at the end should help restricting more the regex. Any thoughts?